hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -273,6 +430,21 @@ def _check_company_id_add_line_task_id(self):
                     _('The Company in the Timesheet Sheet and in '
                       'the Task must be the same.'))
 
+    @api.multi
+    def _get_possible_reviewers(self):
```

Comment:
@mreficent @astirpe to give you better insights: this method has another 6 lines of code specific to validation models.